src/App.tsx

```diff
@@ -1,24 +1,99 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, {
+  useEffect,
+  useState,
+  useMemo,
+  useContext,
+  useRef,
+} from 'react';
+import classNames from 'classnames';
+import './styles/App.scss';
+import { TodoFilter } from './types/TodoFilter';
+import { TodoList } from './components/TodoList';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { getFilteredTodos } from './utils/getFilteredTodos';
+import { CurrentError } from './types/CurrentError';
+import { TodoContext } from './Context/TodoContext';
+import { USER_ID } from './utils/constants';
+import * as todoService from './api/todos';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todoFilter, setTodoFilter] = useState<TodoFilter>(TodoFilter.All);
+
+  const {
+    todos,
+    setTodos,
+    error,
+    setError,
+  } = useContext(TodoContext);
+
+  useEffect(() => {
+    todoService.getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setError(CurrentError.LoadingError);
+        throw new Error();
```

Comment:
You don't need to throw an error in this case, as in this case, you don't catch this error
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/63243136/689391b0-7a3f-453d-b810-e0b193969b96)